integration-tests/src/test/java/oracle/kubernetes/operator/ItImageTool.java

```diff
@@ -0,0 +1,225 @@
+// Copyright (c) 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator;
+
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.Properties;
+import java.util.logging.Level;
+
+import oracle.kubernetes.operator.utils.Domain;
+import oracle.kubernetes.operator.utils.ExecCommand;
+import oracle.kubernetes.operator.utils.ExecResult;
+import oracle.kubernetes.operator.utils.LoggerHelper;
+import oracle.kubernetes.operator.utils.Operator;
+import oracle.kubernetes.operator.utils.TestUtils;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.MethodOrderer.Alphanumeric;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+/**
+ * Simple JUnit test file used for testing Operator.
+ *
+ * <p>This test is used for creating Operator(s) and domain(s) which are managed by the Operator(s).
+ * WebLogic Docker image is created by WebLogic Image Tool
+ */
+@TestMethodOrder(Alphanumeric.class)
+public class ItImageTool extends BaseTest {
+  private static final String TEST_RESOURCE_LOC = "integration-tests/src/test/resources";
+  private static String weblogicImageVersionWIT;
+  private static String weblogicImageNameWIT;
+  private static String weblogicImageTagWIT;
+
+  private static Operator operator;
+  private static Domain domain;
+  private static String domainNS1;
+  private static String testClassName;
+  private static StringBuffer namespaceList;
+
+  /**
+   * This method gets called only once before any of the test methods are executed. It does the
+   * initialization of the integration test properties defined in OperatorIT.properties and setting
+   * the resultRoot, pvRoot and projectRoot attributes.
+   *
+   * @throws Exception exception
+   */
+  @BeforeAll
+  public static void staticPrepare() throws Exception {
+    if (FULLTEST) {
+      testClassName = new Object() {
+      }.getClass().getEnclosingClass().getSimpleName();
+
+      // Determine image name and version to be used
+      // load app props defined
+      LoggerHelper.getLocal().log(Level.INFO, "Loading props from: " + APP_PROPS_FILE);
+      Properties appProps = TestUtils.loadProps(APP_PROPS_FILE);
```

Comment:
You may want to take a look at BaseTest.initialize() method where we load the props file and have getter methods for the properties. Loading props file here is redundant.

Reply:
Yea. I noticed that. However, I need props from this file before BaseTest.initialize()  gets called, plus I don't want to hard-code anything. That's why this call here